internal/fs/fs.go

```diff
@@ -872,16 +895,16 @@ func (fs *fileSystem) unlockAndDecrementLookupCount(in inode.Inode, N uint64) {
 //
 // Typical usage:
 //
-//     func (fs *fileSystem) doFoo() (err error) {
```

Comment:
Looks like some editor formatting changes, please revert.

Reply:
Lint is not complaining, so i feel this is right formatting.